plugins/crate-copy-azure/src/main/java/io/crate/copy/azure/AzureBlobStorageSettings.java

```diff
@@ -60,16 +57,10 @@ public class AzureBlobStorageSettings {
         PROTOCOL_SETTING
     );
 
-    static void validate(Settings settings, boolean read) {
-        List<String> validSettings = Lists.concat(
-            SUPPORTED_SETTINGS.stream().map(Setting::getKey).toList(),
-            read ? COMMON_COPY_FROM_SETTINGS : COMMON_COPY_TO_SETTINGS
-        );
-        for (String key : settings.keySet()) {
-            if (validSettings.contains(key) == false) {
-                throw new IllegalArgumentException("Setting '" + key + "' is not supported");
-            }
-        }
+    /**
+     * Scheme specific validation. Common validation is done by the {@link SchemeSettings}.
+     */
+    static void validate(Settings settings) {
         if (settings.get(SAS_TOKEN_SETTING.getKey()) == null && settings.get(KEY_SETTING.getKey()) == null) {
             throw new IllegalArgumentException("Authentication setting must be provided: either sas_token or key");
```

Comment:
That's what I meant- we cannot have common validation only, there can be something scheme specific.